Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.css

```diff
@@ -0,0 +1,3 @@
+.right{
```

Comment:
Instead of using the custom class `right`, make use of the equivalent bootstrap class `text-right`.